OpenRA.Mods.Common/Widgets/WidgetUtils.cs

```diff
@@ -265,9 +266,18 @@ public static string TruncateText(string text, int width, SpriteFont font)
 			return trimmed;
 		}
 
-		public static Color GetContrastColor(Color fgColor, Color bgDark, Color bgLight)
+		public static LabelWithTooltipWidget TruncatedLabelWithTooltip(LabelWithTooltipWidget label, string text)
```

Comment:
On second thought, `TruncateLabelToTooltip` might be a better name for this.

Reply:
Changed.